Integrations/PhishLabsIOC_EIR/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+- change display name phishlabs eir to phishlabs eir v2
```

Comment:
do we really need changelog here?

Reply:
the old Phishlabs integration is going to be called "Phishlabs EIR" (so maybe we should...)